docusaurus/docs/installation.md

```diff
@@ -146,7 +172,7 @@ yarn install --force
 yarn reassure
 ```
 
-## CI integration
+### Integration
```

Comment:
Why remove CI from title?

Reply:
Because CI Setup is the name of the whole section encompassing these sub-section.

Reply:
<img width="189" alt="Screenshot 2023-02-06 at 16 02 10" src="https://user-images.githubusercontent.com/18323849/217006622-3a04f086-dce1-4733-bc7f-d95f767cf7c5.png">

Like in the docs